datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -726,6 +751,7 @@ private void configureQueryResolvers(final RuntimeWiring.Builder builder) {
             .dataFetcher("browse", new BrowseResolver(browsableTypes))
             .dataFetcher("browsePaths", new BrowsePathsResolver(browsableTypes))
             .dataFetcher("dataset", getResolver(datasetType))
+            .dataFetcher("role", getResolver(roleType))
```

Comment:
I have one final comment. Even though internally the model is Role, let's call the external resolver "externalRole". I didn't realize that we already have another Query called "listRoles", so at least at GraphQL we will need this distinction, even if the underlying PDL models do not have it.

Reply:
(see line 800)